lwt/.merlin

```diff
@@ -1,6 +1,6 @@
 B ../_build/default/lwt
```

Comment:
I am getting used to add them to .gitignore

the less auto-generated noise we have the better I think.